src/test/java/openfasttrack/cli/TestCommandLineInterpreter.java

```diff
@@ -10,12 +10,12 @@
  * it under the terms of the GNU General Public License as
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
- * 
```

Comment:
The License Template contains a space that gets removed by the Eclipse formatter. We have to correct the the template.